fsfuzz

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
is this change necessary? Simple sh should handle this file correctly, can't it?